src/Avalonia.Themes.Fluent/Controls/CaptionButtons.xaml

```diff
@@ -1,72 +1,97 @@
+
+  <ControlTheme x:Key="FluentCaptionButton" TargetType="Button">
+    <Setter Property="Background" Value="#ffe5e5e5" />
+    <!-- Reusing BorderBrush to define pressed background color, as it's not used otherwise -->
+    <Setter Property="BorderBrush" Value="#ffcacaca" />
+    <Setter Property="Foreground" Value="Black"/>
+    <Setter Property="Width" Value="45"/>
+    <Setter Property="VerticalAlignment" Value="Stretch"/>
+    <Setter Property="Template">
+      <ControlTemplate>
+        <ContentPresenter Name="PART_ContentPresenter"
+                          Background="Transparent"
+                          Content="{TemplateBinding Content}"/>
+      </ControlTemplate>
+    </Setter>
+
+    <Style Selector="^:pointerover /template/ ContentPresenter">
+      <Setter Property="Background" Value="{TemplateBinding Background}" />
+    </Style>
+
+    <Style Selector="^:pressed /template/ ContentPresenter">
+      <Setter Property="Background" Value="{TemplateBinding BorderBrush}" />
+    </Style>
+  </ControlTheme>
+  
+  <ControlTheme x:Key="{x:Type CaptionButtons}" TargetType="CaptionButtons">
     <Setter Property="MaxHeight" Value="30" />
     <Setter Property="Template">
       <ControlTemplate>
         <StackPanel Spacing="2" VerticalAlignment="Stretch" TextElement.FontSize="10" Orientation="Horizontal">
-          <StackPanel.Styles>
-            <Style Selector="Panel">
-              <Setter Property="Width" Value="45" />
-              <Setter Property="Background" Value="Transparent" />
-            </Style>
-            <Style Selector="Panel:pointerover">
-              <Setter Property="Background" Value="#7F7f7f7f" />
-            </Style>
-            <Style Selector="Panel#PART_CloseButton:pointerover">
-              <Setter Property="Background" Value="#7FFF0000" />
-            </Style>
-            <Style Selector="Viewbox">
-              <Setter Property="Width" Value="11" />
-              <Setter Property="Margin" Value="2" />
-            </Style>
-          </StackPanel.Styles>
-          <Panel x:Name="PART_FullScreenButton">
-            <Viewbox>
-              <Path Stretch="UniformToFill" Fill="{TemplateBinding Foreground}" />
+          <Button x:Name="PART_FullScreenButton"
+                  Theme="{StaticResource FluentCaptionButton}"
+                  IsVisible="False">
+            <Viewbox Width="11" Margin="2">
+              <Path Name="FullScreenButtonPath"
+                    Stretch="UniformToFill"
+                    Fill="{TemplateBinding Foreground}"
+                    Data="M2048 2048v-819h-205v469l-1493 -1493h469v-205h-819v819h205v-469l1493 1493h-469v205h819z" />
             </Viewbox>
-          </Panel>
-
-          <Panel x:Name="PART_MinimiseButton">
-            <Viewbox>
-              <Path Stretch="UniformToFill" Fill="{TemplateBinding Foreground}" Data="M2048 1229v-205h-2048v205h2048z" />
+          </Button>
+          <Button x:Name="PART_MinimiseButton"
```

Comment:
@danwalmsley I changed these to be buttons, because previously as panels you lost the ability to press and then cancel the press by releasing outside the bounds of the control.